### PR TITLE
Fix/windows build

### LINKED ISSUE #713: Including win32ipc.node into packaged build
We currently have a Node Addon module at app/modules/win32ipc.  

node-gyp properly builds it at creates a build/Release/win32ipc.node  

When working locally this is typically fine to use the absolute path: 

 https://github.com/decred/decrediton/blob/master/app/main.development.js#L306

But this won't work for builds that are used on other machines.  From what I've researched we need to include with node-loader then we can then get a relative path to be used here.  But so far unsuccessful.  

https://stackoverflow.com/questions/32832179/using-webpack-with-nodejs-native-modules

### PR DESCRIPTION
**Pull request by oktapodia**
Fixes #713 appveyor can now build the windows package for us

Still have an issue, the electron app does not close gracefully, the `dcrd` process still running in the background, but think it was already the case, @alexlyp can you confirm please?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,9 @@
       "package.json"
     ],
     "extraResources": [
-      "bin/*"
+      { "from": "bin/dcrd.exe", "to": "bin/dcrd.exe"},
```
Comment (oktapodia) [inline package.json]: Was not working with a wildcard :(

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+install:
+  - ps: Install-Product node 6 x64
+  - git reset --hard HEAD
+  - npm install npm -g
+  - go version
+  - go env
+  - go get -u -v github.com/decred/dcrd
+  - go get -u -v github.com/decred/dcrwallet
+  - go get -u -v github.com/Masterminds/glide
```
Comment (alexlyp) [inline appveyor.yml]: you can remove glide references if you have the dep there.

Comment (oktapodia) [reply appveyor.yml]: But the glide is needed for `dcrd` isn't it?

Comment (alexlyp) [reply appveyor.yml]: derp you're right.  someone is slow with it.... :)

**Pull request merged**